controllers/OrganisasiController.js

```
@@ -1,44 +1,60 @@
-        const organisasi = await Organisasi.create({
-            organisasi
-        });
-        response(res, "Organisasi created successfully", organisasi, 201);
-    } catch (error) {
-        console.error(error);
-        response(res, error.message, null, 500);
-    }
-}
+  try {
+    const organisasi = await Organisasi.create({
+      organisasi,
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)